source/NVDAObjects/window/edit.py

```diff
@@ -372,7 +372,8 @@ def _getTextRange(self,start,end):
 			if self.obj.isWindowUnicode or (res>1 and (buf[res]!=0 or buf[res+1]!=0)): 
 				text=ctypes.cast(buf,ctypes.c_wchar_p).value
 			else:
-				text=unicode(ctypes.cast(buf,ctypes.c_char_p).value, errors="replace", encoding=locale.getlocale()[1])
+				encoding=locale.getlocale()[1]

```

Comment:
I wonder whether we should change this into locale.getpreferredencoding() while at it.